docs/detections/api/rules/rules-api-bulk-actions.asciidoc

```diff
@@ -427,7 +427,18 @@ POST api/detection_engine/rules/_bulk_action
 [discrete]
 ===== Response payload
 
-For `enable`, `disable`, `delete`, `edit`, and `duplicate` actions, a JSON object containing the action's outcome: number of failed and successful outcomes, number of rules, and number of rule objects created, deleted or updated (depending on the action).
+For `enable`, `disable`, `delete`, `edit`, and `duplicate` actions, a JSON object containing the action's outcome:
+
+- `attributes.summary.total`: total number of rules matching the bulk action
+- `attributes.summary.succeeded`: number of successful outcomes (number of rules that were enabled or deleted or updated - depending on the type of action)
+- `attributes.summary.failed`: number of failed outcomes
+- `attributes.summary.skipped`: number of rules that were skipped due to various reasons (see below)
+- `attributes.results.created`: rule objects that were created during the action's execution
+- `attributes.results.updated`: rule objects that were updated during the action's execution
+- `attributes.results.deleted`: rule objects that were deleted during the action's execution
+- `attributes.results.skipped`: rules that were skipped during the action's execution
```

Comment:
Added a few small style edits, and a revision to tighten up the description (it's probably clear enough that "enabled, deleted, or updated" are different types of actions, without needing to point that out).
```suggestion
- `attributes.summary.total`: Total number of rules matching the bulk action
- `attributes.summary.succeeded`: Number of successful outcomes (number of rules that were enabled, deleted, or updated)
- `attributes.summary.failed`: Number of failed outcomes
- `attributes.summary.skipped`: Number of rules that were skipped due to various reasons (explained below)
- `attributes.results.created`: Rule objects that were created during the action's execution
- `attributes.results.updated`: Rule objects that were updated during the action's execution
- `attributes.results.deleted`: Rule objects that were deleted during the action's execution
- `attributes.results.skipped`: Rules that were skipped during the action's execution
```